d1lod/d1lod/sesame/interface.py

```diff
@@ -185,13 +175,20 @@ def insertModel(self):
             print "Attempted to insert a model that was None."
             return
 
+        # checking for all the statements in the current model
+        # if either subject / object / predicate is a blank node - then indicate that the payload contains a blank node
+        blank_node = False
+        for s in self.model:
+            blank_node = self.tripleHasBlankNode(s.subject, s.predicate, s.object)
+            if(blank_node == True):
```

Comment:
Unnecessary parens around conditional:

```
if(blank_node == True):
```

to

```
if blank_node == True:
```

Reply:
Made conditions Python2 compliant.
Commit: https://github.com/ec-geolink/d1lod/pull/48/commits/be01976d391a3f32b6e6b63c6f6dfde7d57988cd